### PR TITLE
Refactory packager tiffdss

### PR DESCRIPTION
**Pull request by jeffsuperglide**
packager refactor with tiffdss C function taking GeoTiff data to DSS record.

### CODE REVIEW

File: async_packager/Dockerfile [hunk 1]
```diff
@@ -1,26 +1,55 @@
...
-WORKDIR /
-COPY packager/requirements.txt ./
-RUN pip3 install --no-cache-dir -r requirements.txt --upgrade
-RUN rm requirements.txt
+RUN make
 
-COPY packager /app/packager
-WORKDIR /app/packager
+# Packager
+FROM osgeo/gdal:ubuntu-small-3.4.3 as packager
```
Comment (adamscarberry) [inline async_packager/Dockerfile]: Might want to bump to 3.5.0

File: async_packager/setup.cfg [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+classifiers =
+    Programming Language :: Python :: 3.6
+    License :: OSI Approved :: MIT License
+
+[options]
+package_dir =
+    = src
+packages = find:
+python_requires = >=3.6
+install_requires =
```
Comment (adamscarberry) [inline async_packager/setup.cfg:20]: Are all these modules the latest stable version?

File: async_packager/setup.cfg [hunk 3]
```diff
@@ -0,0 +1,39 @@
+[metadata]
+name = cumulus-packager
+version = 0.1.0
```
Comment (adamscarberry) [inline async_packager/setup.cfg:3]: I would call this 2.0

File: async_packager/setup.cfg [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+author = USACE RSGIS
+author_email = author@example.com
+description = Packager Implementation
+long_description = file: README.md, LICENSE
+license = MIT
+long_description_content_type = text/markdown
+url = https://docs.rsgis.dev
```
Comment (adamscarberry) [inline async_packager/setup.cfg]: This url looks bogus.  not sure it really matters at this point.

**Review by adamscarberry — COMMENTED** [review]
Overall looks good.  I made a few comments.

File: async_packager/Dockerfile (hunk 1 above)
Comment (jeffsuperglide) [reply async_packager/Dockerfile]: I must have rolled it back at some point but I did have it at 3.5.  There are some dependencies that may need updating too.  Removing from the setup, installing and doing a pip list or freeze would be good to update those requirements.

File: async_packager/setup.cfg (hunk 3 above)
Comment (jeffsuperglide) [reply async_packager/setup.cfg:3]: Cumulus-packager is separate from cumulus-geproc but with similarities.  I'd prefer having one package for both containers; less to maintain.

File: async_packager/setup.cfg (hunk 4 above)
Comment (jeffsuperglide) [reply async_packager/setup.cfg]: More of a placeholder.  Should have a better url.

**Pull request merged**